pages/index.tsx

```diff
@@ -17,7 +21,7 @@ export default function Home() {
       className='overflow-hidden min-h-screen text-center font-sans text-white bg-cover'
       style={{ backgroundImage: "url(/bg.webp)" }}>
       <nav className='text-xl md:text-3xl w-full drop-shadow-lg flex justify-center md:justify-between items-center mb-8 font-header text-left px-4 py-3 font-semibold'>
-        <span>Cal Hacks x MindsDB</span>
+        <img src={images[Math.floor(Math.random()*images.length)]} className="drop-shadow-lg" style={{height: '48px'}} />
```

Comment:
Forgive me if I'm wrong, but won't this build once on server; thus, the same bear gets loaded each time?

Reply:
yeah your right, i realised this after making the PR... i'll make some other alternative tomorrow

Reply:
theres probably some dumb way like a SSR with a little fetch which just returns the image url lol

Reply:
Might want to explore using this - https://nextjs.org/docs/pages/building-your-application/data-fetching/get-server-side-props